Library/Formula/git-when-merged.rb

```diff
@@ -0,0 +1,13 @@
+class GitWhenMerged < Formula
+  desc "Find the where a commit was merged in git."
```

Comment:
Yes, I was originally using the description from the repo but the build complained that it was too long.